### ISSUE TITLE
Illegal rebrand

### ISSUE THREAD

**Issue by vatterspun** [MathewSachin/Captura#405]
Just making you aware I came across what appears to be Captura being sold as a trialware program on the Microsoft Store by some clown named "WonderfulLife"

https://www.microsoft.com/en-us/p/screen-recorder-pro-display-window-region/9nkn36vnk55z?activetab=pivot:overviewtab
https://www.softpedia.com/get/Multimedia/Video/Video-Recording/Screen-Recorder-Pro.shtml

I've reported the issue to Microsoft's Store and Softpedia. 

I'm less surprised that Softpedia missed this just because their database is massive, but disappointed Microsoft didn't run a simple image search for just duplicate images across two sites they own.

**Comment by MathewSachin** [comment]
I recognized from his email on Microsoft store that this person had sent me an email before.

**Jack Fan**
uwpworker@gmail.com

```
He wanted me to take down the source code for Captura.
He would then sell and make money from it. 😮 
In return he would give me $100 a month. (pretty cheap 😏)
```

Like he thinks I'm an idiot 😆.
Captura is, was and will always be completely free and open-source.
I didn't really bother replying to him.

---

Thank you @vatterspun for letting me know.
I've also reported it on Microsoft Store.
I hope this won't last for long.
Anyways, I don't think he would be able to carry the project forward.

**Comment by vatterspun** [comment]
That's very encouraging.  Numerous projects (Fastcopy and Paint.NET come to mind) have seen someone dupe their code and try to sell it and decide being open isn't worth the trouble.  I agree that this is a temporary issue and I'm glad to hear you're sticking with the open thing.

**Comment by MathewSachin** [comment]
To anyone viewing this issue, please report the app on Microsoft store so it will be removed sooner. Thank you for your support.

https://www.microsoft.com/en-us/p/screen-recorder-pro-display-window-region/9nkn36vnk55z?activetab=pivot:overviewtab

**Comment by MathewSachin** [comment]
https://www.microsoft.com/en-us/p/screen-recorder-plus-for-windows-10/9pj1c0rfbzn3?activetab=pivot:overviewtab#

This one seems to be **ScreenToGif** sold by the same person.

@NickeManarin

**Comment by NickeManarin** [comment]
@MathewSachin Thanks, I just reported it now (for some reason, the report tool is kind of broken, even with Firefox).
That's the fourth (I believe so) copycat app being sold at MS Store that I'm aware of.

**Comment by vatterspun** [comment]
> That's the fourth (I believe so) copycat app being sold at MS Store that I'm aware of.

Yikes, that's ... some vetting process they have in place.

In any case, I'm very fond of both ScreenToGif and Captura so I'll definitely keep my eyes open for other copycats.

**Comment by MathewSachin** [comment]
Another one: https://www.microsoft.com/en-in/p/screen-recorder-pro-for-win10/9n27gq6mls5s?cid=msft_web_chart&&activetab=pivot:overviewtab|pivot:reviewstab

**Comment by qayyumabro** [comment]
Just reported on MS store:
"This is the opensource project which this user offering here without original author consent. https://github.com/MathewSachin/Captura

This was reported way back in May 2019 and its still active on MS Store is beyond me. https://github.com/MathewSachin/Captura/issues/405

Please remove this."

**Comment by vatterspun** [comment]
> Just reported this on MS Store

Great work.  Please if you have any kind of Twitter account to call out @Microsoft for this.  I suspect that is a separate unit so painting them with an anti-open source, anti-Github (also Microsoft-owned) brush is likely to see more action.

**Comment by nlebellier** [comment]
still true in 08/04/2020

**Comment by vatterspun** [comment]
Related: #521

**Comment by MathewSachin** [comment]
So, checking back after more than a year, that copy cat app and many others are still up. And they have the features from the latest releases now.
I really feel like my efforts are in vain here and I'm losing interest in the Windows platform as a whole.

Apologies to everyone whose issues/mail/chats were neglected. I was busy with my studies and internships.
More so with mails, I feel like the people contacting me privately can likely be the ones doing this (I'm sorry I don't know who they are, really apologize for not responding).

I know there are some BIG issues in the app, but I can't get motivated enough to solve them due to these things. More so, even though the app is getting popular, I don't have any fellow contributors to ease the load.

I'm moving towards other technology stacks like Android, where I have support from other fellow contributors and people are already recognizing my work. Windows was once a platform of choice for development for me. Sadly it isn't anymore. I'll leave the repo as it is for a few months and then maybe archive it if no one is interested to contribute. I know those issues are very troublesome to fix and much of the code base is a mess written by the inexperienced me.
Since, I'm giving up on it anyway, feel free to fork or sell or do whatever with the code base. It'll still stay open-source here on GitHub.

**Comment by vatterspun** [comment]
>I really feel like my efforts are in vain here 

First, you are not alone in this frustration and Microsoft has done itself no favors after basically driving the unlicensed software prosecution effort with the Business Software Alliance since the early 90s.  Now it seems they don't care at all.  It's a great way to alienate all those people they were trying to bring into the Microsoft ecosystem with a billion dollar Github investment.

Second, many projects have come out and had to deal with rebranding, their code being used in projects, or their licenses being disrespected.  One of my favorites (it's down at the moment), is the FFMPEG "wall of shame" (https://news.ycombinator.com/item?id=790316) for groups that ignored the license terms.  But I've lost track of the number of programs that came out over the years that were just slightly rebranded/modified copies of GIMP who are happy to accept donations.

What's different about the appearance on the Microsoft Store is that it gives the veneer of safety and confidence, and an easy way to accept money.  So projects can quickly appear and start selling.  If one gets shut down, no biggie as the other 3 projects have already made $40 for very little effort.

That said, I hope you haven't lost faith in the open source effort and there are many reasons to keep going with an open model for your future work.  To name a few:

* Open projects gather more interest more quickly
* You tend to see more involvement and more feedback than in closed projects
* More thoughtful participation

People feel a sense of ownership over the programs because they're a sort of community property.  It's definitely why I'm writing this very long post.  There's a sense of permanence because the code could continue; not every open project that closes up shop finds another maintainer but I can definitely say what happened to every closed source tool out there.  And many people in more secure environments can't use anything closed or that goes through some kind of audit.

Finally, I'll note that one more aggressive route you can take is to use GPL licensed software and ask the FSF to intervene on your behalf.  They have a legal team that's definitely embarrassed major companies into compliance.

**Comment by marekr** [comment]
@MathewSachin 
FYI, the easiest answer because even though you licensed it under MIT is to file a DMCA claim with Microsoft because you still own the copyright.

Microsoft DMCA Form
https://www.microsoft.com/en-us/concern/dmca

Unlike the Microsoft Store report option that goes to some generic call center dumpster, the DMCAs do go to an actual human review legal department.

**Comment by atiq4pk** [comment]
I have reported this app on above link and get the following answer:

![image](https://user-images.githubusercontent.com/14157000/88453887-7dcfd980-ce84-11ea-86ba-16f13fd29143.png)

This is very strange that after knowing the fact, they are just closing their eyes.

**Comment by marekr** [comment]
@atiq4pk No, it's about legal cover your ass. If they were to take it down due to a third party complaining, but it turns out that the app was really authorized by the owner to distribute (not in this case of course), Microsoft exposes themselves to damages due to loss of sales, etc. If the owner complains, then Microsoft isn't going to be held liable for anything, the issue remains squarely between the two parties. This is just how America, the land of lawyers works.

But as you can see, theres actually a rather prompt human review for things coming into their DMCA system (as legally required). @MathewSachin should be able to submit it, distinctly state he is owner of the copyright and the distribution is violating his license and be done with it.

**Comment by MathewSachin** [comment]
@vatterspun I'm not leaving open-source. Most of my personal projects would still be open-source. I love to get feedback from the community. It's just that I've lost interest in Windows as a platform to develop for due to these troubles. I'm switching to Android right now.
I'm pretty sure that app is being sold with FFMpeg bundled in it, which violates their license too. But, I don't have the time/willingness to go behind these legal things.
I can only pity the people who buy apps from WIndows Store. Most of the software there is copy-cat crap.

@marekr Since its MIT License, those people would just put the copyright notice buried somewhere in their UI and with that they can claim that they're not violating the license. Submitting a copyright claim from my side won't help due to that. I don't want to change to GPL, since it takes away the freedom from the software. FFMpeg has more of a chance to claim the copyright violation here because of their license.

@atiq4pk Really appreciate that you reported to them.
